dtfs-central-api/src/services/repositories/utilisation-reports-repo/utilisation-reports-repo.ts

```diff
@@ -32,15 +32,15 @@ export const saveUtilisationReportDetails = async (
   };
 
   const utilisationReportDetailsCollection = await db.getCollection(DB_COLLECTIONS.UTILISATION_REPORTS);
-  const updatedResult = await utilisationReportDetailsCollection.updateOne(
```

Comment:
I think a better name would be something like `updateUtilisationReportDetailsWithUploadDetails` - thoughts?

Also, we can just pass the fields we want to set rather than the whole object (in fact, it makes more sense to do this as we only want to update the fields related to updating reports rather than overwriting the original properties with the same values!)

Reply:
Sounds good to me 👍